templates/addons/azure-cni-v1/kustomization.yaml

```diff
@@ -2,5 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - https://raw.githubusercontent.com/Azure/azure-container-networking/v1.5.3/hack/manifests/cni-installer-v1.yaml
-
+  - https://raw.githubusercontent.com/Azure/azure-container-networking/4034aad0d7085f2a9e96cce6d2b50b81ea9ec900/hack/manifests/cni-installer-v1.yaml
```

Comment:
So your fix merged in azure-container-networking *after* the v1.15.16 tag?

Shoud we wait for v1.15.17 so both these references can point to an official release?